samples/CMakeLists.txt

```diff
@@ -45,6 +49,11 @@ endif()
 
 add_executable(amber ${AMBER_SOURCES})
 target_include_directories(amber PRIVATE "${CMAKE_BINARY_DIR}")
+
+if (EXISTS ${PROJECT_SOURCE_DIR}/third_party/lodepng)
+  target_compile_definitions(amber PRIVATE LODEPNG_EXIST=1)
```

Comment:
This would then move to the main CMakeLists file similar to `add_definitions(-DAMBER_ENABLE_SHADERC=$<BOOL:${AMBER_ENABLE_SHADERC}>)
`